scripts/updateSources.js

```diff
@@ -120,36 +110,10 @@ async function main() {
     if (/OverlayIT\.java$/.test(source)) {
       content = content.replace(/\/overlayselfattached/,`/${wc}$&`)
     }
-
-    // Accordion: Match textfield version
-    if (/AccordionInTemplate\.java$/.test(source)) {
-      content = content.replace(/@NpmPackage.*/,textFieldVersion);
-    }
```

Comment:
This fix is no longer relevant after Vaadin 20 where we aligned web components version.